Makefile

```diff
@@ -119,9 +129,12 @@ mixedindentlint: node_modules/mixedindentlint
 	@$(RECORD_ENV) $@
 
 # generate the client-side `config` js file
-$(CLIENT_CONFIG_FILE): .env config/$(CALYPSO_ENV).json config/client.json client/config/regenerate.js
+$(CLIENT_CONFIG_FILE): .env config/$(CALYPSO_ENV).json config/client.json client/config/regenerate.js client/lib/user/shared-utils.js
 	@$(NODE) client/config/regenerate.js > $@
 
+client/lib/user/shared-utils.js: server/user-bootstrap/shared-utils.js
+	@ln -s $(realpath $<) $@
+
```

Comment:
I think a better solution is to make `client/lib/user/shared-utils.js` something like this:

`module.exports = require( 'server/user-bootstrap/shared-utils' );`